tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -107,6 +107,7 @@ def test_spectre_meltdown_checker_on_host(spectre_meltdown_checker):
     utils.run_cmd(f"sh {spectre_meltdown_checker} --explain")
 
 
+@pytest.mark.nonci
```

Comment:
Why did you squash the backport commit?
https://github.com/firecracker-microvm/firecracker/commit/720141fb38f4b29257200566e5901656e3091750

Reply:
Yes, let's keep other commits separate and properly attributed with `git cherry-pick -x`

Reply:
It is ideal to have each commit work. WIthout the `@pytest.mark.nonci` changes the commit doesn not work.

Reply:
The commit cannot be cherry picked without changes.

Reply:
I've split the commit.

Reply:
Splitting the commit also works, but "cherry-picking the commit and then modifying it with `git commit --amend`" could be better, because the commit message is (almost) identical with additional information (the upstream commit hash and the explanation why we use `nonci` instead of `no_block_pr` (the lack of `no_block_pr` on v1.4 branch).)

Reply:
nitpick: it would be good to reference the original commit with a SHA1, but I won't block the PR on this.

Reply:
+1 to Pablo, but I also won't block the PR.